docs/guide/blog/comment-blog.md

```diff
@@ -0,0 +1,548 @@
+---
+description: Add comment on Blog
+order: 2
+---
+
+# Add comments to Blog
+
+In this tutorial, you will create a new message module called comment. The module will let you read and write comments to the existing blog blockchain.
+
+You can only add comments to a post which is no older than 100 Blocks. 
+
+**Note:** This value has been hard coded to a low number for rapid testing. You can increase it to a greater number to achieve longer period of time before commenting is stopped.
+
+### Prerequisites:
+
+- This tutorial is an extension of previously written blog tutorial. Make sure you complete that first before proceeding with this tutorial.
+- This tutorial also assumes basic knowledge of blog tutorial implementation.
+- Make sure you are inside the `blog` directory created in the previous blog tutorial.
+
+## Create a new message called comment
+
+To create a new message module for comment, use the `message` command:
```

Comment:
There is no "message module".

Reply:
I'll rephrase as: To create a new message, use the `message` command:

Reply:
That works.